src/database/schema/ChannelListSchema.ts

```diff
@@ -6,6 +6,7 @@ import ChannelListMemberSchema from './ChannelListMemberSchema';
 import UserSchema from './UserSchema';
 import {AnonUserInfo} from '../../../types/service/AnonProfile.type';
 import {AnonymousPostNotification} from '../../../types/repo/AnonymousMessageRepo/AnonymousPostNotificationData';
+import {BetterSocialChannelType} from '../../../types/database/schema/ChannelList.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BetterSocialChannelType` is a good change as it enhances type safety by replacing the generic `string` type with a more specific one. This will help prevent potential bugs and improve code readability.
```diff
+ import {BetterSocialChannelType} from '../../../types/database/schema/ChannelList.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->